src/seeds/db-seeds.ts

```diff
@@ -5,14 +5,12 @@ import { generateHash } from 'provider/hash-provider';
 import { connectToDatabase } from 'setup-server';
 import { getRepository } from 'typeorm';
 
-async function seedUsers() {
-  envConfig();
-  await connectToDatabase();
+export const seedUsers = async (numberOfUsers: number) => {
```

Comment:
Sugiro deixar aqueles 50 como valor default desse parâmetro. Assim vc pode continuar usando sem passar, se quiser.
```suggestion
export const seedUsers = async (numberOfUsers = 50) => {
```